pkgs/applications/networking/instant-messengers/ripcord/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, stdenv, fetchurl, writeScript, appimageTools, buildFHSUserEnv, makeFontsConf,
+  gsettings-desktop-schemas, gtk3, twemoji-color-font }:
+
+let
+  pname = "ripcord";
+  version = "0.4.23";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://cancel.fm/dl/Ripcord-${version}-x86_64.AppImage";
+    sha256 = "0395w0pwr1cz8ichcbyrsscmm2p7srgjk4vkqvqgwyx41prm0x2h";
+    name = "${pname}-${version}.AppImage";
+  };
+
+  fontsConf = makeFontsConf {
+    fontDirectories = [
+      twemoji-color-font
+    ];
+  };
```

Comment:
doing this just feels very hacky. I don't have any better suggestion right now though.

Reply:
I agree, but having spoken to the developer, he has no plans to fix this bug, because in his eyes it's a bug in the font rendering, not his code, and doesn't want to work around it. For the purposes of making it usable I think this is the best option.